presidio-api/cmd/presidio-api/main.go

```diff
@@ -73,6 +76,11 @@ func main() {
 	setupHTTPServer(settings.WebPort, settings.LogLevel)
 }
 
+// Note:
+// 	 When modifying the API: Like adding or removing end-points, updating parameters, or updating responses,
+// 	 remember to make the relevant modifications in the files under the /docs package, and then re-generate
```

Comment:
can this be done automatically in CI? or at least have some validation during CI that it did. otherwise it won't happen.

Reply:
I didn't find a way to update the docs files automatically when modifying the end points. 
Also validation seems impossible.